packages/appkit/src/library/react/index.ts

```diff
@@ -141,16 +141,21 @@ export function useAppKitState() {
     throw new Error('Please call "createAppKit" before using "useAppKitState" hook')
   }
 
-  const [state, setState] = useState(modal.getState())
+  const [state, setState] = useState({ ...modal.getState(), initialized: false })
 
   useEffect(() => {
-    const unsubscribe = modal?.subscribeState(newState => {
-      setState({ ...newState })
-    })
-
-    return () => {
-      unsubscribe?.()
+    if (modal) {
+      setState({ ...modal.getState() })
```

Comment:
This line is going to override `initialized: false` right?

Reply:
yes but since it will execute client side, that's ok. After this, we subscribe to the state and it will auto update